community/cypher/docs/cypher-docs/src/docs/dev/ql/delete/index.asciidoc

```diff
@@ -2,9 +2,11 @@
 = Delete
```

Comment:
Perhaps we should add a "FORCE DELETE" section as well so that it shows up in the TOC.